content/build-apps/running-localnet.draft

```diff
@@ -12,9 +13,9 @@ You can start up a full mix network, including mixnodes, and a gateway on your l
 
 First, download, build and run the [Nym directory server](https://github.com/nymtech/nym-directory). This step will soon go away, as the directory server is about to be retired in the {{< param stable >}}  release. But for now it's necessary.
 
-{{% notice warning %}}
+{{< attention title=" " >}}
 The local network **must** be able to access a directory server running on `localhost`! Make sure you install and run the directory server.
```

Comment:
There's no directory server anymore (users would need to run a valdiator and valdiator api). Probably the only practical way to get a full local setup running in under a few hours would be to use Bogdan's Docker containers. However I'd be reluctant to point people at those in the docs until we see how brittle they turn out to be. 
For the moment I would just remove this whole section until we get it right.

Reply:
this is a leftover draft from before i joined the team - I can just remove this page entirely if you want?